yb_migrate/cmd/importData.go

```diff
@@ -380,8 +379,11 @@ func checkPrimaryKey(tableName string) bool {
 		table = strings.Split(tableName, ".")[0]
 	}
 
+	/* currently object names for yugabytedb is implemented as case-sensitive i.e. lower-case
+	but in case of oracle exported data files(which we use for to extract tablename)
+	so eg. file EMPLOYEE_data.sql -> table EMPLOYEE which needs to converted for using further */
 	checkPKSql := fmt.Sprintf(`SELECT * FROM information_schema.table_constraints
-	WHERE constraint_type = 'PRIMARY KEY' AND table_name = '%s' AND table_schema = '%s';`, table, schema)
+	WHERE constraint_type = 'PRIMARY KEY' AND table_name = '%s' AND table_schema = '%s';`, strings.ToLower(table), schema)
```

Comment:
This will break case-sensitive PG migration.

Reply:
Just pushed the updated. Have a look @amit-yb